yacc/reader.c

```diff
@@ -282,12 +282,12 @@ static void process_open_curly_bracket(FILE *f) {
         }
     }
 
-    if (In_bitmap(caml_ident_start, *idcptr) || *idcptr == '|')
+    if ((*idcptr >= 'a' && *idcptr <= 'z') || *idcptr == '|')
```

Comment:
It would be nice to have a function `is_caml_quoted_string_id_body` to factor this check on `'a'..'z'`. Note also that I think your test is slightly wrong, underscores are also accepted (see [the manual](https://v2.ocaml.org/releases/4.14/htmlman/lex.html#sss:stringliterals)).